sandbox/check_schema.py

```diff
@@ -130,7 +130,7 @@ def __print_patch_response(key: str, response):
 
     # GET all the existing modules of Yangcatalog
     LOGGER.info('Fetching all of the modules from API')
-    all_existing_modules = fetch_modules(LOGGER)
+    all_existing_modules = fetch_modules(LOGGER, config=config)
```

Comment:
Why config cannot be loaded directly in `fetch_modules` method?
Like why we need to pass it as an argument when "default" argument value can be used?

Reply:
Just to not create a new config instance every time, there's no deeper meaning behind this. The only other case is when some variable in the config can be changed, like in tests, so we can control this